CONTRIBUTING.md

```diff
@@ -307,16 +307,23 @@ If the commit reverts a previous commit, it should begin with `revert:`, followe
 
 #### Type
 
-Must be one of the following:
+The following types will cause a version bump:
 
-- **feat**: A new feature
-- **fix**: A bug fix
+- **fix**: Patches a bug in the code and directly corresponds to the **PATCH**
+- **feat**: Describes a new feature and corresponds to the **MINOR**
+- **BREAKING CHANGE**: a commit that has a footer `BREAKING CHANGE:`, or appends a `!` after the type/scope, introduces a breaking API change (correlating with **MAJOR** in semantic versioning).
+
+> Appending a `!` and/or a `BREAKING CHANGE:` footer to **ANY TYPE** will denote a **BREAKING CHANGE** and will cause a **MAJOR** bump
+
+The following types will _**not**_ cause a version bump:
+
+- **build**: Changes that affect the build system or external dependencies
+- **ci**: Changes to our CI configuration files and scripts
 - **docs**: Documentation only changes
-- **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-- **refactor**: A code change that neither fixes a bug nor adds a feature
 - **perf**: A code change that improves performance
```

Comment:
Performance fixes won't trigger a patch release?

Reply:
Im just going off what i've read on https://conventionalcommits.org/

Reply:
Can you add a comment then? Something like `Based on [conventional commits guidance]()`

Reply:
Eh close enough
https://github.com/webex/webex-js-sdk/pull/1787/files#diff-6a3371457528722a734f3c51d9238c13R286